doc/howto_release.md

```diff
@@ -150,6 +151,7 @@ git tag -n --sort=-taggerdate
 Now push the tag upstream - this will trigger the automated workflows linked to tags:
 
 ```bash
+# TODO: how to enforce the right release branch?
```

Comment:
@wenzeslaus How does the tag magic work for `release_branch_8_2`? With `git push upstream` I guess it would land in `main` and not the branch...

Reply:
> The tag is made on the branch - a specific commit actually - not pushed to the branch, so what matters is the `git tag` step.

Yes, I see (still learning this part of git). From related the [SO](https://stackoverflow.com/a/27154277/452464) answer which you indicated I read:

_Specifying a branch name as the tag target defaults to that branch's most recent commit; e.g.:_

    git tag v0.1.0  develop   # tags HEAD of 'develop' branch

_(As others have noted, you can also specify a commit ID explicitly as the tag's target.)_

So this should translate into

    git tag v7.8.8RC3 releasebranch_8_2   # tags HEAD of 'releasebranch_8_2' branch

... right? But then `push` to where?

(Importantly, the instructions here shall become fail-safe to also be able to successfully release even when tired ;-) )

Reply:
> BTW, do you want to name next release branch release_branch_8_3, not releasebranch_8_3? :-)

Sorry, typo...